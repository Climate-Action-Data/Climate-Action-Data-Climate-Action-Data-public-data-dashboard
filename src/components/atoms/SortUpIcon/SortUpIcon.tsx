import { FC } from 'react'
import { Icon, IconProps } from '@chakra-ui/react'

export const SortUpIcon: FC<IconProps> = (props: IconProps) => {
  const actualProps = {
    ...props,
    width: props?.width ?? `8px`,
    height: props?.height ?? `4px`,
  }
  return (
    <Icon viewBox="0 0 8 4" {...actualProps}>
      <path d="M3.99911 0.1923L0.191406 4H7.80681L3.99911 0.1923Z" fill="currentColor" />
    </Icon>
  )
}
