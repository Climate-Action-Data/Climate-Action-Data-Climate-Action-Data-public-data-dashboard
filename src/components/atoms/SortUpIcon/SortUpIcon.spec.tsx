'use strict'

import { render } from '@testing-library/react'
import { SortUpIcon } from './SortUpIcon'

it(`renders correctly`, () => {
  const { container } = render(<SortUpIcon />)
  expect(container).toMatchSnapshot()
})
