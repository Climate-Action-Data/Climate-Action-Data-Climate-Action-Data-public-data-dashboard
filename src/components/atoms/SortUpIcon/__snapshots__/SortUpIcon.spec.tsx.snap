// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <svg
    class="chakra-icon css-c0air1"
    focusable="false"
    viewBox="0 0 8 4"
  >
    <path
      d="M3.99911 0.1923L0.191406 4H7.80681L3.99911 0.1923Z"
      fill="currentColor"
    />
  </svg>
</div>
`;
